internal/lookup/root/root.go

```diff
@@ -79,6 +81,34 @@ func (r *Driver) Libraries() lookup.Locator {
 	)
 }
 
+// LibCudaPath returns the path to libcuda.so.RM_VERSION in the driver installation.
+func (r *Driver) LibCudaPath() (string, error) {
+	candidates, err := r.Libraries().Locate("libcuda.so.1")
+	if err != nil {
+		return "", err
+	}
+
+	pattern := "*.*"
+	var matched []string
+	for _, candidate := range candidates {
+		// It is not guaranteed that candidate is a resolved symlink.
+		resolved, err := symlinks.Resolve(candidate)
+		if err != nil {
+			continue
```

Comment:
Can we consider logging the error messages?

Reply:
Done.